airbyte-webapp/src/services/connectorBuilder/ConnectorBuilderStateService.tsx

```diff
@@ -36,7 +36,8 @@ const useJsonManifest = () => {
 
 const useSelected = () => {
   const { jsonManifest } = useJsonManifest();
-  const streamListRead = useListStreams({ manifest: jsonManifest });
+  const configJson = useConfig();
```

Comment:
```suggestion
  const { configJson } = useConfig();
```
You'll need to have the curly braces here in order to pull the `configJson` value out of the `useConfig()` response

Reply:
oops yeah, i had the deference before but was testing something w/o will add back. in thanks for pointing it out!